cfg/CFG.cc

```diff
@@ -22,12 +22,46 @@ BasicBlock *CFG::freshBlock(int outerLoops, int rubyBlockId) {
     return r.get();
 }
 
+LocalRef CFG::enterLocal(core::LocalVariable variable) {
+    if (!localVariableToLocalRef.contains(variable)) {
```

Comment:
this can be done in a single lookup, not two.

Reply:
Yup; that's the plan. I did it this way because `localRef.exists()` returns false for `noVariable` which I want to be able to stick into the array at the 0th entry. There are a few ways to fix that, but I wanted to get unblocked and benchmark.